components/raftstore/src/store/worker/stats_monitor.rs

```diff
@@ -331,9 +337,32 @@ impl Recorder {
 
     fn sample(samples: &mut Vec<Sample>, key_range: &KeyRange) {
         for mut sample in samples.iter_mut() {
-            // we think key range is full open interval, such as (a,b) or (a,"").
-            let order_start = sample.key.cmp(&key_range.start_key);
-            let order_end = sample.key.cmp(&key_range.end_key);
+            if key_range.start_key == key_range.end_key {
+                // point
+                if key_range.start_key.is_empty() || key_range.start_key == sample.key {
```

Comment:
It's not full open interval now.
Suppose this workload: 100 * ('a', 'a') + 100 * ('b', 'b').
There won't be any split key found.